libs/assets/powershell/iis_global_rewrite.ps1

```diff
@@ -46,6 +53,18 @@ if($RewriteType -eq "url"){
     Set-WebConfigurationProperty –pspath $path –Filter "$filter/action" –Name "url" –Value $RedirectUrl;
 } elseif ($RewriteType -eq "farm"){
     Set-WebConfigurationProperty –pspath $path –Filter "$filter/action" –Name "url" –Value "http://$RedirectUrl/{R:1}";
+    if($ConditionalFirst -eq $true){
+        Set-WebConfigurationProperty –pspath $path –Filter "$filter/conditions" -Name "logicalGrouping" -Value "MatchAll";
+        Add-WebConfigurationProperty –pspath $path –Filter "$filter/conditions" –Name "." –Value @{input="{HTTP_HOST}";matchType="Pattern";pattern=$ConditionalPattern;negate="false";};
+    }
+    if($ConditionalSecond -eq $true){
+        Set-WebConfigurationProperty –pspath $path –Filter "$filter/conditions" -Name "logicalGrouping" -Value "MatchAll";
+        Add-WebConfigurationProperty –pspath $path –Filter "$filter/conditions" –Name "." –Value @{input="{REQUEST_URI}";matchType="Pattern";pattern ='*/api/*';negate="true";};
+        Add-WebConfigurationProperty –pspath $path –Filter "$filter/conditions" –Name "." –Value @{input="{REQUEST_URI}";matchType="Pattern";pattern ='*/admin/*';negate="true";};
+        Add-WebConfigurationProperty –pspath $path –Filter "$filter/serverVariables" –Name "." –Value @{name='HTTP_X_FORWARDED_HOST';value='{HTTP_HOST}';replace='true';};
+        Add-WebConfigurationProperty –pspath $path –Filter "$filter/serverVariables" –Name "." –Value @{name='HTTP_X_FORWARDED_SCHEMA';value='https';replace='true';};
+        Add-WebConfigurationProperty –pspath $path –Filter "$filter/serverVariables" –Name "." –Value @{name='HTTP_X_FORWARDED_PROTO';value='https';replace='true';};
+    }
```

Comment:
This is a generic IIS server rewrite script. There are hard-coded strings in this section 

This was a really lazy way to do it.